limit_to_employees/models/res_partner.py

```diff
@@ -0,0 +1,18 @@
+from odoo import api, models
+from odoo.osv import expression
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.model
+    def get_mention_suggestions(self, search, limit=8):
+        """ override the default method and customize the search filter"""
+        search_dom = expression.OR([[('name', 'ilike', search)], [('email', 'ilike', search)]])
+        search_dom = expression.AND([[('active', '=', True)], search_dom])
```

Comment:
From what I see here, you look up if there is a user like the search input and if they are active.
So, in this context: Employees are activated partners with a user attached? Is this the logic you like to express?

Reply:
It shows the Internal users in dropdown those are active. Yes, in Employee form its related user is set, so in dropdown it shows only  Employees. But if Internal user employee is not created, then also that user is showing in that dropdown.